src/modules/esl-popup/core/esl-popup.ts

```diff
@@ -41,24 +56,31 @@ export class ESLPopup extends ESLToggleable {
   @prop() public closeOnOutsideAction = true;
 
   public connectedCallback() {
-    this.$arrow = this.querySelector('.esl-popup-arrow');
-
     super.connectedCallback();
+
+    setTimeout(() => {
+      this.$arrow = this.querySelector('span.esl-popup-arrow');
+    });
```

Comment:
Why was it moved to a separate task?
Mb we need to mark the whole connected callback with @ready decorator instead?

Reply:
It's a "horrible piece of shit" (c).
I added this crutch and completely forgot.
Fixed